src/server/components/formRunner/helpers.ts

```diff
@@ -87,30 +79,16 @@ export async function generateFormRunnerWebhookData(
 }
 
 export async function parseJsonFormData(
-  listType: string,
-  isUnderTest: boolean = false
+  listType: "funeralDirectors" | "lawyers" | "translatorsInterpreters"
 ): Promise<Array<Partial<FormRunner.Question>>> {
-  /**
-   * TODO:- Ideally we can do a require.resolve(..) which will look in the current directory for the target, then in the parent etc
-   * so that we don't need the isUnderTest flag. However, I suspect an issue to do with webpack is preventing us from
-   * doing this properly. See branch `origin/fix/containers` rev 1e76...6bb.
-   * For now, we need to keep ./forms-json in sync with /docker/apply/forms-json.
-   * I have tried doing a babel/tsc/webpack/jest moduleNameMapping change but it is still causing errors.
-   * Giving up. Enjoy
-   */
-  let baseDir = __dirname.replace("dist", "dist/src/server/components/formRunner");
-  if (!baseDir.includes("dist")) {
```

Comment:
fwiw this was copying to dist like `dist/src/server/components/formRunner`. No longer necessary. It is included in the ts/babel build process, similar to how you may use a package. `import lawyersForm from @forms/lawyers` for example.